src/app/workflow/workflow.component.ts

```diff
@@ -413,15 +413,21 @@ export class WorkflowComponent extends Entry implements AfterViewInit, OnInit {
       this.labelsEditMode = true;
       return;
     }
+    const value = this.labelFormControl.value;
+    if ((value || '').trim()) {
+      this.workflowEditData.labels.push(value.trim());
+    }
     // the edit object should be recreated
-    if (this.workflowEditData.labels !== 'undefined') {
+    if (this.workflowEditData.labels !== undefined) {
```

Comment:
I wonder about the value of this `if` statement in general -- if `this.workflowEditData.labels` really is `undefined` and `value` is truthy, it would have already failed on line 418. But I assume nobody (or at least I don't) knows this code that well, so it's probably safest not to tinker with it.

Heh, also, since the labels always seems to be an array, it "worked" before because an array does not equal the string value `"undefined"`.